scripts/glean.py

```diff
@@ -134,11 +132,30 @@ def _set_definition(self, full_defn: dict):
         self.definition["origin"] = full_defn[self.ORIGIN_KEY]
         self.definition["date_first_seen"] = self.definition_history[-1]["dates"]["first"]
 
-    def _set_description(self, definition: dict):
-        if "description" in definition:
-            self.description = definition["description"]
-        else:
-            self.description = None
+
+class GleanTag(GleanObject):
+    """
+    Represents an individual Glean Tag, as defined by probe scraper
+    """
+
+    def __init__(self, identifier: str, definition: dict):
+        self.identifier = identifier
+        self._set_definition(definition)
+        self.description = self.definition_history[0].get("description")
+
+    def _set_definition(self, full_defn: dict):
+        self.definition_history = list(
+            sorted(
+                full_defn[self.HISTORY_KEY],
+                key=lambda x: datetime.fromisoformat(x["dates"]["last"]),
+                reverse=True,
+            )
+        )
+
+        # The canonical definition for up-to-date schemas
+        self.definition = self.definition_history[0]
+        self.definition["name"] = full_defn[self.NAME_KEY]
```

Comment:
looking at the screenshot, I saw that the new "science" tag for burnham is lower case. Since I'm used to seeing fenix tags being upper case and lower case for origin labels, UI-wise, I didn't immediately associate "science" to be a tag. I think it's probably a good idea to keep the formatting consistent here, i.e. capitalize the tag label where we can.

Reply:
Interesting, do you think we should suggest that at the glean level? I think it will be hard to enforce conventions on tags. Another alternative would be to add a 🏷️ emoji to the end of tags (just like we do with origin labels and the 📦 emoji). I'd probably do this as a seperate PR

Reply:
tbh this is not a big issue, it's probably just my own mental model (uppercase = tags, lowercase = origin) and no one else's. I still think we should experiment with adding a tag emoji to make it clearer that a label is a tag. Will do it in a separate PR.